netket/operator/_discrete_operator_jax.py

```diff
@@ -181,12 +181,15 @@ def n_conn(self, x, out=None) -> np.ndarray:
 
         Returns:
             array: The number of connected states x' for each x[i].
-
         """
+        _, mels = self.get_conn_padded(x)
+        nonzeros = jnp.abs(x) > 0
```

Comment:
shouldn't we add an explicit cutoff here? otherwise this might be a bit too sensitive to floating point precision of specific hardware

Reply:
that could work, but I'd give the option to pass a custom cutoff, in most cases you want to ignore things that are larger than floating point zero !

Reply:
sure thing, what should be the default value of the cutoff? note n_conns interface itself does not have this kwargs defined so i'm guessing for api consistency u want to have this as an operator property?

Reply:
yes indeed we should maybe add this to the abstract class interface as a property .  issue is that this is a breaking change, especially if we set something like 10^-8 as cutoff

Reply:
yes. just to clear atm w/o this patch the implementation is incorrect. so i'm guessing anything that fixes that w/o a breaking change would be good to have.

Reply:
yes agreed, let's merge this and think about the cutoff later, thanks !